### ISSUE TITLE
Docs claim case sensitive is default

### ISSUE THREAD

**Issue by vojtechkral** [TheNeikos/envious#35]
Hi, just noticed the README says

> _Note:_ The environment variables **are** case sensitive by default! This can be modified using the [`Config::case_sensitive`] method.

... this doesn't seem to be true anymore as of v 0.2

**Comment by TheNeikos** [comment]
You're right. It was changed in https://github.com/TheNeikos/envious/commit/685b19fdd0954edd8492a420013617a6e42d3d16

The behaviour is correct, and the docs need to be fixed.
